machine.go

```diff
@@ -886,11 +911,21 @@ func (m *Machine) GetMetadata(ctx context.Context, v interface{}) error {
 // parameters of the partialDrive.
 func (m *Machine) UpdateGuestDrive(ctx context.Context, driveID, pathOnHost string, opts ...PatchGuestDriveByIDOpt) error {
```

Comment:
This `opts` will be ignored on the PUT path as well, but nobody is using that on GitHub.

https://github.com/search?q=UpdateGuestDrive&type=Code

Reply:
I've reverted the change.